keyboards/lily58/keymaps/via/rules.mk

```diff
@@ -1,2 +1,5 @@
 VIA_ENABLE = yes
 OLED_DRIVER_ENABLE = yes
+MOUSEKEY_ENABLE = yes
+EXTRAKEY_ENABLE = yes
+GRAVE_ESC_ENABLE = yes
```

Comment:
```suggestion
```
This is enabled by default.

Reply:
@nblyumberg I do see your point as well regarding consistency of `rules.mk` in the VIA keymap—most of them contain only `VIA_ENABLE = yes`. The reason is that most VIA-only users keyboard (KBDFans for example) have mouse and extra keys are enabled by default on the top level `rules.mk`.

Lily58 probably falls in to the ergo-enthusiast category. Its VIA `rules.mk` already has `OLED_DRIVER_ENABLE = yes` so I thought this would be the appropriate location to add mouse/extrakeys support for HEX files that will be built for VIA's site.

If I'm not wrong, all VIA hex files are rebuilt _periodically_ at the same time—you can see the [last-modified date consistency.](https://github.com/the-via/firmware)